server/src/controllers/Auth.controller.js

```diff
@@ -68,4 +69,59 @@ export default class AuthController {
       message: 'User registered successfully'
     });
   }
+
+  /**
+   * @method login
+   *
+   * @param {object} req request
+   * @param {object} res response
+   *
+   * @returns {object} returns status code, data and message properties
+   */
+  static login(req, res) {
+    const { email, password } = req.body;
+
+    for (let i = 0; i < users.length; i++) {
+      if (email === users[i].email) {
+        const verifyPassword = comparePassword(password, users[i].password);
+
+        if (!verifyPassword) {
+          return res.status(404).json({
+            status: 404,
```

Comment:
The status code is wrong. Failed authentications should return `401, Unauthorized`. 404 means it wasn't found. In this case, the email was found but the password was not incorrect. Please correct as in,
```suggestion
            status: 401,
```

Reply:
done...